user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md

```diff
@@ -35,15 +35,13 @@ If you have the Owner or Admin role on dataminer.services for a DMS, you can ena
 
 1. In the pane on the left, under *DataMiner Systems*, expand the DataMiner System, and select the *Settings* page.
 
-1. Toggle the *Remote Access* settings to *On* or *Off*, depending on whether you want this to be enabled or not.
+1. Select how remote access should be limited:
 
-   If the main *Remote Access* setting is enabled, the following settings are also available:
+   - If you want remote access to be disabled completely, switch off the main *Remote Access* toggle button.
 
-   - *DataMiner Cube (desktop app)*: Enable or disable remote access to the DataMiner System(s) via the desktop app.
+   - If you want remote access to be enabled, switch on the main *Remote Access* setting. If it should only be enabled for specific DataMiner features, below this setting, switch off the features for which remote access should be disabled.
```

Comment:
don't we want to list & explain the sub remote access settings like cube, user defined apis, web apps?

Reply:
I left these out to keep the text simple and more update-proof, since users will see the different options in front of them anyway. Would you prefer to explicitly mention them in the documentation after all?

Reply:
I think it's good if they search/browse in the docs and can find out here that they can disable specific things. Also the extra information explaining each setting, that in the UI is behind a tooltip(hover), shown here wouldn't hurt. If we add settings we indeed need to add them here (is expected i think)